.github/workflows/CI.yml

```diff
@@ -124,6 +124,57 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  book-tests:
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version:
+          - '1.10'
+          - 'pre'
+          - 'nightly'
+        group:
+          - 'book'
```

Comment:
You don't need to set the group here, and could instead hardcode it below. This would also remove some duplication in the job title `booktest (1.10, book)`.
But no high priority